reference/opt/shaders-hlsl/comp/access-chains.comp

```diff
@@ -1,5 +1,3 @@
-static const uint3 gl_WorkGroupSize = uint3(1u, 1u, 1u);
```

Comment:
@HansKristian-Work Could it be that these test shaders haven't been updated in a while? Because when I run `update_test_shaders.sh` with a recent KhronosGroup/SPIRV-Cross clone that I have locally (from Sep. 5th), all `(1u, 1u, 1u)` and `uint3(1u)` variants of `gl_WorkGroupSize` disappear just like this with PR.

Reply:
The dependencies were recently updated. Try ./checkout_glslang_spirv_tools.sh and build new deps.